app/src/main/java/com/example/weatherandroidapp/viewmodels/WeatherGridViewModel.kt

```diff
@@ -0,0 +1,7 @@
+package com.example.weatherandroidapp.viewmodels
+
+import androidx.lifecycle.ViewModel
+
+class WeatherGridViewModel : ViewModel() {
+    // TODO: Implement the ViewModel
+}
```

Comment:
is this neccesary?

Reply:
No, but I had created the empty weather grid fragment for the interaction, it seemed good to me that it has the respective view model, and I will use it in the future